fdbserver/LogSystemPeekCursor.actor.cpp

```diff
@@ -187,10 +187,24 @@ void ILogSystem::ServerPeekCursor::advanceTo(LogMessageVersion n) {
 	}
 }
 
+void padReplyToEnd(ILogSystem::ServerPeekCursor* self, const TLogPeekReply& res) {
+	if (!res.lockedEnd) {
+		return;
+	}
+	if (self->end.version == std::numeric_limits<Version>::max()) {
+		TraceEvent("GotLockedEndMax").detail("S", res.end);
```

Comment:
Can we ever get here? It appears to me if "lockedEnd" is set then "self->end.version" must have been set to the recovery version?